.github/workflows/cmake.yml

```diff
@@ -93,18 +105,24 @@ jobs:
         with:
             build_mode: "Release"
 
-    call-release-cmake-cygwin:
-        name: "CMake Cygwin Workflows"
-        uses: ./.github/workflows/cygwin-cmake.yml
-        with:
-            build_mode: "Release"
-
     call-release-cmake-xpr:
         name: "CMake TestExpress Workflows"
         uses: ./.github/workflows/testxpr-cmake.yml
 
-    call-release-cmake-julia:
-        name: "CMake Julia Workflows"
-        uses: ./.github/workflows/julia-cmake.yml
+#    call-release-cmake-julia:
```

Comment:
Should decide whether we're going to leave the Julia tests disabled from develop or re-enable them and bring in the needed fix